core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -134,9 +134,11 @@ public List<AclRequests> selectFilteredAclRequests(
     }
 
     for (AclRequests row : aclListSub) {
+
       Integer teamId;
       String rowRequestOperationType = row.getRequestOperationType();
       if (isApproval) {
+        teamSelected = showRequestsOfAllTeams ? null : teamSelected;
```

Comment:
can null here lead to nullpointer?

Reply:
No the only use of it after this has a !=null check so it should be safe. (line 158)